README.md

```diff
@@ -1 +1,9 @@
 # tarantool-qa
+
+Repository for QA activities and issues of the [tarantool/tarantool](https://github.com/tarantool/tarantool) project.
+
+Main target of this repository to detach internal QA activities from [Tarantool issues](https://github.com/tarantool/tarantool/issues) interesting to Tarantool users.
+This repository uses for issues on:
```

Comment:
Please, split paragraphs with an empty line: this way it looks better as in text as well as rendered.

Reply:
Sure, done.